versions/sponge.json

```diff
@@ -1,5 +1,6 @@
 {
     "versions": [
+        "7.0.0-SNAPSHOT",
```

Comment:
I would go and add 6.0.0 officially and 6.1.0-snapshot officially, since those have been released already to the public.